server/src/db/seeders/ReviewSeeder.js

```diff
@@ -0,0 +1,32 @@
+import { Review } from "../../models/index.js";
+
+class ReviewSeeder {
+  static async seed() {
+    const reviewsData = [
+      {
+        movieId: 1,
+        userId: 1,
```

Comment:
we cannot assume that there is a movie or user with a specific id - you need to query the movie and user records that you want to reference and use their actual `id` property